frontend/Gemfile

```diff
@@ -7,3 +7,4 @@ git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
 # gem "rails"
 
 gem "fastlane", "~> 2.194"
```

Comment:
is there a reason why we use another fastlane version then in `frontend/ios/Gemfile`